docs/mindsdb-docs/docs/sql/tutorials/predicting_house_price_tutorial.md

```diff
@@ -0,0 +1,110 @@
+## Predicting house price
+In this tutorial you will learn how to predict house price on basis of different parameter like condition,bedrooms,bathrooms,area etc. We will be using mysql database for this purpose and kc_house_data as dataset.
+The process will be like:
+
+ 1. Cleaning data and deleting unnecessary columns.
+ 2. Starting MySQL main server to create databases and tables.
+ 3. Starting mindsdb server to access gui and its database.
+ 5. Starting another MySQL server on port 47735 with username mindsdb
+ 7. Predicting using our predictor.
+
+These are our 5 step to add ML prediction model layer on any mysql database. 
+So let get started.
+
+## Cleaning data:
+Simply install database and clean it using excel ,your data should look like this:
+![excle_img](https://user-images.githubusercontent.com/59201258/141510855-b8e2325a-361c-44c4-9fd8-8adfb6729602.PNG)
+
+
+## Starting mysql main server
+So now time to start root server, open cmd and type:
+
+    mysql -h 127.0.0.1 --port 3306 -u root -p
+   
+   Now create our database:
+   
+
+    CREATE DATABASE housedata;
+Now create the table:
+
+    USE housedata
+    CREATE TABLE house_pricing_data;
```

Comment:
I think here we will need the column names too or it will work without them?

Reply:
no it will work fine because we are importing csv ,you can check on your side to!